src/database/seeders/20210120052931-userRoles.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-unused-vars */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.bulkInsert('userRoles', [{
+      name: 'ADMIN_ROLE',
```

Comment:
Can you seed all roles that are needed in the whole project (other than **ADMIN_ROLE** and **SUPER_ADMIN**)?